### PR TITLE
Add target for running phantom tests

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- run `make start-phantom`
- visit `http://0.0.0.0:8888`
- you should get test output in your browser

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4180/
Test PASSed.

File: Makefile [hunk 1]
```diff
@@ -364,6 +365,10 @@ test-mocha-karma: gui
 start-karma:
 	$(NODE_MODULES)/.bin/karma start karma.conf.js
 
+.PHONY: start-phantom
```
Comment (hatched) [inline Makefile]: This doesn't actually run phantom, just python pserve with the test/index.html

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4181/
Test PASSed.

**Comment by hatched** [comment]
:+1: :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**